### PR TITLE
Update making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst

### PR DESCRIPTION
**Pull request by camow7**
Added steps required to make this tutorial work with the widely used Edimax Nano USB WiFi adapter (EW-7811Un).

### CODE REVIEW

File: dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst [hunk 1]
```diff
@@ -416,22 +416,41 @@ Update hostapd
...
 
 ::
 
     wget http://adafruit-download.s3.amazonaws.com/adafruit_hostapd_14128.zip
+    
+To get the new version of hostapd for the Edimax 7811 Dongle type the following command:
+
+::
+
+    wget http://www.daveconroy.com/wp3/wp-content/uploads/2013/07/hostapd.zip
```
Comment (magicrub) [inline dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst]: Dave Conroy? Sounds fishy

Comment (camow7) [inline dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst]: I followed this tutorial to get it going: http://www.daveconroy.com/turn-your-raspberry-pi-into-a-wifi-hotspot-with-edimax-nano-usb-ew-7811un-rtl8188cus-chipset/

Perhaps there is a way to host his zip file on the JDrone Servers?

**Comment by rmackay9** [comment]
I'd like to point out that apsync now has a beta image for the RPI3 (although it works on the RPI2 as well).
http://ardupilot.org/dev/docs/apsync-intro.html

File: dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst [hunk 2]
```diff
@@ -416,22 +416,41 @@ Update hostapd
 --------------
 
 The hostapd version installed by apt-get does not fully support the WiFi
-dongle we are using so we need to update it to a later version. Lets get
-the new version of hostapd by typing the follwing command:
+dongle we are using so we need to update it to a later version. 
+
+To get the new version of hostapd for the Adafruit Dongle type the following command:
```
Comment (khancyr) [inline dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst]: I think this is wrong, it is the contrary, you need to step back to older version that support that chip.

File: dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst (hunk 1 above)
Comment (khancyr) [inline dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst]: or give link to to officiel hostapd from edimax ... 2017 and those still aren't fix ... nice raspi communauty....

File: dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst (hunk 2 above)
Comment (camow7) [inline dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst]: Yep, you're right. I have modified it to say "later version" instead.

**Comment by camow7** [comment]
@rmackay9 will the async tutorial just end up replacing the WiFi MavLink bridge tutorial?

**Comment by magicrub** [comment]
@rmackay9 can you answer @camow7 's question? If the answer is yes then we should close this PR

**Comment by rmackay9** [comment]
@camow7, to answer the question, "no", we won't remove the mavlink wifi bridge instructions even though the easier-to-use APSync image is available.

**Comment by rmackay9** [comment]
I'm a little worried about the link to daveconroy so I think we should fix that before I merge this.

**Comment by rmackay9** [comment]
I think i'll close this for now.

I wonder if most users want to attach an external dongle to an RPI in any case these days because the RPI3 (and RPI4 I think) come with a wifi chip on the board.

**Comment by camow7** [comment]
Yeah this is all out-of-date now any way. Better ways to solve most of these problems.

**Pull request closed without merging**